src/Controller/ConfigController.php

```diff
@@ -341,7 +341,7 @@ public function getAction(Request $request, Service $graphQlService): JsonRespon
             $config['schema']['specialEntities'] = [];
         }
 
-        $specialSettings = ["document", "document_folder", "asset", "asset_folder", "object_folder"];
+        $specialSettings = ["document", "document_folder", "asset", "asset_folder", "asset_listing", "object_folder"];
```

Comment:
translation key missing. probably something like "plugin_pimcore_datahub_graphql_special_asset_listing"

Reply:
besides that I think we should disable the configuration option other than "read"

Reply:
translation key: done
I think we should keep this for now, I am planning to create a separate PR to allow to extend that from outside DataHub, I can refactor that with that then. WDYT?

Reply:
Sounds good!